### PR TITLE
nvmeof-recoverer: fix `reconstructCRUSHMap` and `runNvmeoFJob()` to work correctly

### PR DESCRIPTION
**Pull request by cheolho-kang**
**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: pkg/operator/ceph/nvmeof_recoverer/clustermanager/handler.go [hunk 1]
```diff
@@ -34,18 +34,18 @@ import time
...
-	result = subprocess.run(['nvme', 'list', '-o', 'json'],
-							stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-	devices = json.loads(result.stdout)
-	devices = {device['DevicePath'] for device in devices['Devices']}
-	return devices
+    result = subprocess.run(['nvme', 'list', '-o', 'json'],
+                            stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    devices = json.loads(result.stdout)
+    devices = {device['DevicePath'] for device in devices['Devices']}
+    return devices
```
Comment (myoungwon) [inline pkg/operator/ceph/nvmeof_recoverer/clustermanager/handler.go:41]: ?

Comment (myoungwon) [reply pkg/operator/ceph/nvmeof_recoverer/clustermanager/handler.go:41]: 두개로 분리 필요

Comment (cheolho-kang) [reply pkg/operator/ceph/nvmeof_recoverer/clustermanager/handler.go:41]: 수정하였습니다.
handler.go PR과 controller PR을 분리하는게 나을 것 같다는 말씀이실까요?

Comment (myoungwon) [reply pkg/operator/ceph/nvmeof_recoverer/clustermanager/handler.go:41]: indentation과 그 외 내용을 분리하는게 좋을 것 같습니다.

Comment (cheolho-kang) [reply pkg/operator/ceph/nvmeof_recoverer/clustermanager/handler.go:41]: 아! 넵 수정하였습니다

**Pull request merged**